README.md

```diff
@@ -1,52 +1,86 @@
+
+## Overview
+I Hate Money is a web application made to ease shared budget management. It keeps track of who bought what, when, and for whom; and helps to settle the bills.
+
+**Shipped version:** 4.1.3
+
+## Screenshots
+
+![](sources/Screenshots.png)
+
+## Demo
+
+* [Official demo](https://ihatemoney.org/)
+
+## Configuration
+
+How to configure this app: From an admin panel, a plain file with SSH, or any other way.
+
+## Documentation
+
+ * Official documentation: Link to the official documentation of this app
+ * YunoHost documentation: If specific documentation is needed, feel free to contribute.
+
+## YunoHost specific features
 
 Backs on MySQL database, the identifiers are per-project, not per-user, so no
-way to do advanced SSO integration with yunohost accounts.
+way to do advanced SSO integration with YunoHost accounts.
 
 The behaviour is either:
 
 - **non-public app**:
-  - yunohost login required
+  - YunoHost login required
   - per-project identifiers required
-  - any yunohost user with authorized access to the app can create a new
+  - any YunoHost user with authorized access to the app can create a new
     project).
 - **public app** :
-  - no yunohost login required
+  - no YunoHost login required
   - per-project identifiers required
   - any visitor can create a new project.
 
-Update
-------
+#### Multi-user support
+
+* Are LDAP and HTTP auth supported?
+* Can the app be used by multiple users?
+
+#### Supported architectures
+
+* x86-64 - [![Build Status](https://ci-apps.yunohost.org/ci/logs/ihatemoney%20%28Apps%29.svg)](https://ci-apps.yunohost.org/ci/apps/ihatemoney/)
+* ARMv8-A - [![Build Status](https://ci-apps-arm.yunohost.org/ci/logs/ihatemoney%20%28Apps%29.svg)](https://ci-apps-arm.yunohost.org/ci/apps/ihatemoney/)
+
+## Limitations
 
-To update the app, use:
+* Any known limitations.
 
-`sudo yunohost app upgrade ihatemoney -u https://github.com/YunoHost-Apps/ihatemoney_ynh`
-  
+## Additional information
 
-Maintainer
-----------
+* Other info you would like to add about this app.
 
-I rely on
-[ihatemoney official releases](https://github.com/spiral-project/ihatemoney/releases)
-(tarballs). Don't hesitate to pull-request this repo if I missed one :-).
+## Links
 
+ * Report a bug: https://github.com/YunoHost-Apps/ihatemoney_ynh/issues
+ * App website: https://ihatemoney.org/
+ * Upstream app repository: https://github.com/spiral-project/ihatemoney/
+ * YunoHost website: https://yunohost.org/
 
-Ihatemoney license
-------------------
+---
 
-> The code is distributed under a BSD beerware derivative: if you meet the
-> people in person and you want to pay them a craft beer, you are highly
-> encouraged to do so.
+## Developer info
 
-[Full license text](https://github.com/spiral-project/ihatemoney/blob/master/LICENSE)
+Please send your pull request to the [testing branch](https://github.com/YunoHost-Apps/ihatemoney_ynh/tree/testing).
```

Comment:
there is no such thing as a testing branch here, so let's not say that in readme :-).

Reply:
It maybe a good idea to create a testing branch... for testing

Reply:
I'd prefer not to discuss organizational/workflow stuff in a doc modification :-)